clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractStickyAssignor.java

```diff
@@ -58,7 +58,7 @@ public abstract class AbstractStickyAssignor extends AbstractPartitionAssignor {
 
     // Keep track of the partitions being migrated from one consumer to another during assignment
     // so the cooperative assignor can adjust the assignment
-    protected Map<TopicPartition, String> partitionsTransferringOwnership = new HashMap<>();
+    public Map<TopicPartition, String> partitionsTransferringOwnership = new HashMap<>();
```

Comment:
Instead of making this public can you actually make it private and just add a getter method that returns it? We don't want any class fields being modifiable from the outside

Reply:
that make sense, but set to private would result in many modifications in ``org.apache.kafka.clients.consumer.internals.AbstractStickyAssignorTest``, I think you mean set protected is enough?  changed to protected now.